js/main.js

```diff
@@ -18,38 +18,89 @@ function renderCountry(country) {
   $flag.classList.add('flag');
-  $country.append($flag);
+  $countryF.append($flag);
 
   const $flagImg = document.createElement('img');
   $flagImg.src = country.flags.png;
   $flagImg.alt = country.flags.alt;
+  $flag.append($flagImg);
 
-  const $name = document.createElement('h2');
-  $name.textContent = country.name.common;
+  const $nameF = document.createElement('h2');
+  $nameF.textContent = country.name.common;
+  $countryF.append($nameF);
 
-  const $line = document.createElement('hr');
+  const $lineF = document.createElement('hr');
+  $countryF.append($lineF);
 
-  const $capital = document.createElement('h3');
-  $capital.textContent = country.capital;
+  const $capitalF = document.createElement('h3');
+  $capitalF.textContent = country.capital;
+  $countryF.append($capitalF);
 
-  const $region = document.createElement('h3');
-  $region.textContent = country.region;
+  const $regionF = document.createElement('h3');
+  $regionF.textContent = country.region;
+  $countryF.append($regionF);
 
-  const $population = document.createElement('h3');
-  $population.textContent = formatPopulation(country.population);
+  const $populationF = document.createElement('h3');
+  $populationF.textContent = formatPopulation(country.population);
+  $countryF.append($populationF);
+
+  // Back Side
+  const $countryB = document.createElement('div');
+  $countryB.classList.add('country', 'country-back');
+  $card.append($countryB);
+
+  const $nameB = document.createElement('h2');
+  $nameB.textContent = country.name.common;
+  $countryB.append($nameB);
+
+  const $lineB = document.createElement('hr');
+  $countryB.append($lineB);
+
+  const $capitalB = document.createElement('h3');
+  $capitalB.innerHTML = '<span>Capitol: </span>' + country.capital[0];
+  $countryB.append($capitalB);
+
+  const $regionB = document.createElement('h3');
+  $regionB.innerHTML = '<span>Region: </span>' + country.region;
+  $countryB.append($regionB);
+
+  const $populationB = document.createElement('h3');
+  $populationB.innerHTML = '<span>Population: </span>' + country.population.toLocaleString();
+  $countryB.append($populationB);
+
+  const $subRegion = document.createElement('h3');
+  $subRegion.innerHTML = '<i class="fa-solid fa-map-pin"></i> ' + country.subregion;
+  $countryB.append($subRegion);
+
+  const $money = document.createElement('h3');
+  $money.innerHTML = '<i class="fa-solid fa-money-bill"></i> ' + Object.keys(country.currencies);
+  $countryB.append($money);
+
+  const arrayLanguage = Object.values(country.languages);
+  const $language = document.createElement('h3');
+  $language.innerHTML = '<i class="fa-solid fa-language"></i> ';
+  for (let i = 0; i < arrayLanguage.length && i < 4; i++) {
+    $language.innerHTML += Object.values(arrayLanguage)[i];
+    if (i !== 3 && i !== arrayLanguage.length - 1) {
+      $language.innerHTML += ', ';
+    }
+  }
+  if (arrayLanguage.length > 4) {
+    $language.innerHTML += ' + more';
```

Comment:
You don't want to use `innerHTML`. This opens you up into security issues by injection attacks. By using `innerHTML` you allow anyone to input any code into an element, so it can allow people to inject their own scripts and whatnot, which allows for malicious attacks to potentially occur.

You can try using this instead https://developer.mozilla.org/en-US/docs/Web/API/Element/setHTML